ci/dependency-update/Jenkinsfile

```
@@ -90,6 +90,11 @@ pipeline {
 		// Run at least once per week, in case of snapshot updates.
 		cron '@weekly'
 	}
+	parameters {
+		choice(name: 'UPGRADE_JOB', choices: ['all'] + configurations.keySet() as String[], defaultValue: 'all', description: 'Select which upgrade jobs to run. `All` will include any upgrade job that does not require building Hibernate ORM from sources.')
```

I think we use the term "update" everywhere else... ?

```suggestion
		choice(name: 'UPDATE_JOB', choices: ['all'] + configurations.keySet() as String[], defaultValue: 'all', description: 'Select which update jobs to run. `All` will include any update job that does not require building Hibernate ORM from sources.')
```

Of course if you think we should use "upgrade" I'm fine with that, but we should do it consistently.

---

Also you might want to update the documentation if you change the way the `requiresLocalOrmBuild` boolean works (see other comments).